tests/fsharp/test-framework.fs

```diff
@@ -114,6 +114,21 @@ module Commands =
         Directory.CreateDirectory path |> ignore
         path
 
+    let dotnet workDir exec (_: FilePath) flag srcFiles =
+        let args = (sprintf "%s %s" flag (srcFiles |> Seq.ofList |> String.concat " "))
+
+        ignore workDir 
+        exec "C:\Program Files\dotnet\dotnet.exe" args
```

Comment:
THis path should not be hardwired

Reply:
Is it possible to get this path from the environment variables?
@forki any ideas from FAKE?

Reply:
I think you have `dotNetExe` don't you

Reply:
The variable is unfortunately incorrect, it's pointing to my knowledge to the wrong path.